tests/safeds/exceptions/test_unknown_column_name_error.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+from safeds.exceptions import UnknownColumnNameError
+
+
+@pytest.mark.parametrize(
+    ("column_names", "similar_columns", "expected_error_message"),
+    [
+        (["column1", "column2"], [], "Could not find column(s) 'column1, column2'."),
+    ],
+    ids=["empty_list"]
+)
+def test_empty_similar_columns(column_names, similar_columns, expected_error_message):
+    error = UnknownColumnNameError(column_names, similar_columns)
+    assert str(error) == expected_error_message
```

Comment:
```suggestion
@pytest.mark.parametrize(
    ("column_names", "similar_columns", "expected_error_message"),
    [
        (["column1", "column2"], [], r"Could not find column\(s\) 'column1, column2'."),
    ],
    ids=["multiple_columns_no_suggestions"]
)
def test_empty_similar_columns(column_names: list[str], similar_columns: list[str], expected_error_message: str):
    with pytest.raises(UnknownColumnNameError, match=expected_error_message):
        raise UnknownColumnNameError(column_names, similar_columns)
```

Reply:
I just made some changes and resolved linter errors. I guess it's similar to yours. See last commit =)